### PR TITLE
created gigs form

### PR DESCRIPTION
**Pull request by vibhuti-p**

### CODE REVIEW

File: app/views/gigs/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <%= f.input :description %>
+  <%= f.input :price_basic %>
+  <%= f.input :content_basic %>
+  <%= f.input :price_standard %>
+  <%= f.input :content_standard%>
+  <%= f.input :price_premium %>
+  <%= f.input :content_premium %>
+  <%= f.input :address %>
+  <%= f.input :picture %>
+  <%= f.submit  "Creat a gig", class: "btn btn-primary"%>
```
Comment (gohyifan) [inline app/views/gigs/_form.html.erb]: Typo error: Change to "Create Gig" instead.

File: app/views/gigs/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<%= simple_form_for new_gig do |f| %>
```
Comment (gohyifan) [inline app/views/gigs/_form.html.erb]: Add hidden field user_id

File: app/views/gigs/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<h1>Create gig</h1>
```
Comment (gohyifan) [inline app/views/gigs/edit.html.erb]: Should be 'Update Gig' instead.

File: app/views/gigs/show.html.erb [hunk 4]
```diff
@@ -1,2 +1,6 @@
 <h1>Gigs#show</h1>
+banner
```
Comment (gohyifan) [inline app/views/gigs/show.html.erb]: Comment out sudo code

File: app/views/gigs/show.html.erb [hunk 5]
```diff
@@ -1,2 +1,6 @@
...
+<h2>@gig.title</h2>
+<p>@gig.description</p>
+packages (render)
```
Comment (gohyifan) [inline app/views/gigs/show.html.erb]: Comment out pseudo code

Comment (vibhuti-p) [reply app/views/gigs/show.html.erb]: done

File: app/views/gigs/show.html.erb (hunk 4 above)
Comment (vibhuti-p) [reply app/views/gigs/show.html.erb]: done

File: app/views/gigs/edit.html.erb (hunk 3 above)
Comment (vibhuti-p) [reply app/views/gigs/edit.html.erb]: ok

File: app/views/gigs/_form.html.erb (hunk 1 above)
Comment (vibhuti-p) [reply app/views/gigs/_form.html.erb]: ok

File: app/views/gigs/_form.html.erb (hunk 2 above)
Comment (vibhuti-p) [reply app/views/gigs/_form.html.erb]: ok

**Pull request merged**